ka/mutablestore.hpp

```diff
@@ -2,7 +2,7 @@
 #define KA_MUTABLESTORE_HPP
 #pragma once
 #include <functional>
-#include <boost/config/suffix.hpp>
+#include <boost/config/detail/suffix.hpp>
```

Comment:
This file should not be included, could you try removing it completely and see if it compiles for you ?
```suggestion
```

Reply:
It actually does, so I removed it. Thanks.